.github/workflows/macos.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}-latest
```

Comment:
I think the problem is here... macos-14-latest doesn't make sense to me

Reply:
Yep, that's what I get for copy/pasting YAML files.  I wish GitHub would tell me the "runs-on" doesn't make sense.